docs/05-ingest.md

```diff
@@ -141,6 +141,10 @@ imported ([see here](06-status.md#reinitialise-a-table)).
 
 There are several stacks that allow data to be imported using the bulk import process:
 
+- `EmrServerlessBulkImportStack` - this causes an EMR Serverless application to be created when the Sleeper instance is deployed. 
+  This is the default EMR Bulk Import Stack. The advantage of using EMR Serverless is that when there are no bulk import jobs 
+  the applications stops with no wasted compute. The startup of the application is greatly reduced compared to standard EMR. 
+  This stack is experimental.
```

Comment:
What makes it experimental?

Reply:
@gaffer01 said it is experimental until it offers all of the features that standard EMR offers e.g. job platformSpec config